packages/example-addon/package.json

```diff
@@ -21,8 +21,10 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-babel": "^6.16.0"
+    "@esri/hub-events": "^1.11.1",
```

Comment:
should `@esri/hub-events` be a peerDep too?

Reply:
it could be if we envision that it would be used outside of this addon. We could just decide to make all hub.js and rest-js packages `peerDependencies`.